examples/social/src/state.rs

```diff
@@ -2,11 +2,11 @@
 // SPDX-License-Identifier: Apache-2.0
 
 use linera_sdk::views::{CustomMapView, LogView, ViewStorageContext};
-use linera_views::views::{GraphQLView, RootView};
+use linera_views::views::RootView;
 use social::{Key, OwnPost};
 
 /// The application state.
-#[derive(RootView, GraphQLView)]
+#[derive(RootView, async_graphql::SimpleObject)]
```

Comment:
As with the `GuardedView`s above, I prefer to refer to it qualified here — since ‘simple object’ isn't really enough to tell the casual reader what's going on (that it's a *GraphQL* object).